common/openpype_common/distribution/addon_distribution.py

```diff
@@ -0,0 +1,267 @@
+import os
+from enum import Enum
+from abc import abstractmethod
+import attr
+import logging
+import requests
+import platform
+import shutil
+
+from common.openpype_common.distribution.file_handler import RemoteFileHandler
```

Comment:
You can use relative import here
```suggestion
from .file_handler import RemoteFileHandler
```

Or this (which would break current tests):
```
from openpype_common.distribution.file_handler import RemoteFileHandler
```

Reply:
Ok.